Wikipedia/Code/WMFDatabaseHouseKeeper.swift

```diff
@@ -2,9 +2,11 @@ import Foundation
 
 @objc class WMFDatabaseHouseKeeper : NSObject {
     
-    @objc func performHouseKeepingOnManagedObjectContext(_ moc: NSManagedObjectContext)  throws {
+    // Returns deleted URLs
+    @objc func performHouseKeepingOnManagedObjectContext(_ moc: NSManagedObjectContext) throws -> [URL] {
```

Comment:
Just a note for future - if this object grows any further I'd like to refactor it to use a command pattern (or something).